.circleci/continue_config.yml

```diff
@@ -0,0 +1,498 @@
+version: 2.1
+
+orbs:
+  gcp-cli: circleci/gcp-cli@2.2.0
+
+parameters:
```

Comment:
Mind documenting how these workflow params are used? They probably are, just not in an obvious way (ex: `<< pipeline.parameters.some-param >>`)

Reply:
Ah- these are unused now. The initial approach I took required them (using the path-filtering orb) and I forgot to delete them.

Reply:
These are deleted